transports/sql/index.md

```diff
@@ -81,6 +81,12 @@ Each endpoint should use a dedicated SQL Server principal with `SELECT` and `DEL
 
 partial: alwaysencrypted
 
+### Retries
+
+When an exception occurs during message handling, the transaction is already in doubt and must be rolled back. Failure info is stored in memory, and can't influence the next step for that message (immediate retry, delayed retry, or forward to the error queue) until the next time it is processed on that node.
```

Comment:
Wouldn't this only be for sends atomic with receive and transaction scope? When using receive only, and not use connection sharing there would be no need to fully rollback the transaction before it can do an immediate retry.

Not sure how the current receive strategy for receive only behaves.

Maybe all strategies use the same retry model. If so, discard my comment.

Reply:
SendsAtomic and ReceiveOnly use the same receiving strategy and [roll back the transaction on error](https://github.com/Particular/NServiceBus.SqlServer/blob/master/src/NServiceBus.Transport.SqlServer/Receiving/ProcessWithNativeTransaction.cs#L48-L52).